### ISSUE TITLE
adding a button of restoring session in "New WIndow"

### ISSUE THREAD

**Issue by Adi2612** [bhavinjawade/Session-House-Chrome-Extension#2]
@bhavinjawade  
When we click on restore  particular session ,all the tabs opened at that time will be automatically closed and  all the tabs of that particular session will be opened. 
So, We can have a new button of "Restoring Sessions and  Opening it in New Window" . So that present important tabs will not be removed.

**Comment by bhavinjawade** [comment]
Right, this feature should be there.
Adding Restore session as a dropdown with 2 options: 
1) Restore in same window.
2) Restore in a new Window.
Thanks for bringing this to attention. Are you taking up this issue?

**Comment by Adi2612** [comment]
> Right, this feature should be there.
> Adding Restore session as a dropdown with 2 options:
> 
> 1. Restore in same window.
> 2. Restore in a new Window.
>    Thanks for bringing this to attention. Are you taking up this issue?

Yup , I will take this issue and I also wanna implement a great UI ( issue : #1 ) for this extension.

**Comment by bhavinjawade** [comment]
Great !

**Comment by bhavinjawade** [comment]
This is still an open issue.
